pkg/cli/helm/helm.go

```diff
@@ -136,3 +178,13 @@ func runUpgrade(upgradeClient *helm.Upgrade, releaseName string, helmChart *char
 	}
 	return err
 }
+
+func UnwrapAll(err error) error {
+	for {
+		unwrapped := errors.Unwrap(err)
+		if unwrapped == nil {
+			return err
+		}
+		err = unwrapped
+	}
```

Comment:
This is to unwrap until you can't find any more errors, right?

Reply:
exactly

Reply:
I'm not sure if there's a better way to do this though

Reply:
what's the problem you're trying to solve by doing this?

Reply:
It looks like you're doing an `errors.As` and then ignoring the result ....

Reply:
yep this is exactly what I was doing. no idea how I missed that functionality for errors.As()....